db/migrate/20240104144005_add_status_to_friendships.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToFriendships < ActiveRecord::Migration[7.1]
+  def change
+    add_column :friendships, :mutual, :boolean, default: false, null: false
```

Comment:
Column to easily verify if a friendship is mutual or not without checking if the inverse friendship exists (i.e., without querying the database a second time).